puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_rpm.yaml

```diff
@@ -1,55 +1,13 @@
-# Builds a single Foreman (sub)project or dependency from the foreman-packaging
-# repository, using Koji.
-#
-# See https://projects.theforeman.org/projects/foreman/wiki/RPM_Packaging#Project-sources-version-3
-# or the help text on the build page for usage.
 - job:
     name: packaging_build_rpm
+    project-type: pipeline
+    sandbox: false
```

Comment:
Is there a reason we can't sandbox?

Reply:
I remember in the past when I was converting to pipeline jobs `sandbox` wasn't what I thought it meant and it ended up making things not work.

Reply:
well [this](https://docs.openstack.org/infra/jenkins-job-builder/project_pipeline.html?highlight=sandbox) isn't helpful:

```
sandbox (bool): If the script should run in a sandbox (default false)
```